### PR TITLE
Guidance around opening links in new tabs

### PR DESCRIPTION
**Pull request by marteki**
Updates the Design Manual to detail the situations when it's more desirable to open links in a new tab.

## Additions

- Detail out behaviors for when to open links in new tabs
- Include wording around code to prevent security problems
- Include guidance and direction for screen reader accessibility

## Review

- @sonnakim 
- @ielerol 
- @Scotchester 
- @cfarm 
- @StephanieLevy 

[Preview this PR without the whitespace changes](?w=0)

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: page-components/links.md [hunk 1]
```diff
@@ -34,6 +34,25 @@ Links lead users to a different page or further information. In contrast, <a hre
   </ul>
 </div>
 
+## Behavior
+
+### Default links
+In general, links should default to opening in the same page or browser tab.
+
+### Opening a link in a new tab
+Links should open in a new tab in interactions where users enter data or make
```
Comment (Scotchester) [inline page-components/links.md]: "in interactions" is somewhat awkward phrasing. "in situations", perhaps?

File: page-components/links.md [hunk 2]
```diff
@@ -34,6 +34,25 @@ Links lead users to a different page or further information. In contrast, <a hre
...
+selections that would be lost if they left the page and used the back button to
+come back to the page. This includes interactive tools, search filters, and
```
Comment (Scotchester) [inline page-components/links.md]: "and used the back button to come back to the page" seems superfluous.

File: page-components/links.md [hunk 3]
```diff
@@ -34,6 +34,25 @@ Links lead users to a different page or further information. In contrast, <a hre
...
+forms where the user has to submit and enter information.
```
Comment (Scotchester) [inline page-components/links.md]: "submit and enter information" is redundant, or at the very least backwards.

File: page-components/links.md [hunk 4]
```diff
@@ -34,6 +34,25 @@ Links lead users to a different page or further information. In contrast, <a hre
...
+
+- Add `target="_blank" rel="noopener noreferrer"` to direct these links to
+securely open in a new tab.
+- Add screenreader-only text to warn users with visual impairments that this
+will open in a new tab to meet <a
+href="https://www.w3.org/TR/WCAG20-TECHS/G201.html">WCAG guideline 3.2 that
+webpages should work in a predictable way</a>.
```
Comment (Scotchester) [inline page-components/links.md]: Could we use a standard Markdown link here, please?

File: page-components/links.md [hunk 5]
```diff
@@ -34,6 +34,25 @@ Links lead users to a different page or further information. In contrast, <a hre
...
+
+### Opening a link in a new tab
+Links should open in a new tab in interactions where users enter data or make
+selections that would be lost if they left the page and used the back button to
+come back to the page. This includes interactive tools, search filters, and
+forms where the user has to submit and enter information.
+
+- Add `target="_blank" rel="noopener noreferrer"` to direct these links to
+securely open in a new tab.
+- Add screenreader-only text to warn users with visual impairments that this
```
Comment (Scotchester) [inline page-components/links.md]: We should provide a code example for this, as well.

File: page-components/links.md [hunk 6]
```diff
@@ -162,6 +181,12 @@ Lorem ipsum dolor sit amet. Vestibulum orci ante, sagittis quis dolor a, fringil
 
 ---
 
+#### Links used in interactive tools, forms, and search filters
+These links should open in a new tab. Set them to `target="_blank"
+rel="noopener noreferrer"`
```
Comment (Scotchester) [inline page-components/links.md]: Missing period at the end.

Comment (Scotchester) [inline page-components/links.md]: We should probably mention the screenreader-only text here, too, yes?

File: page-components/links.md (hunk 3 above)
Comment (StephanieLevy) [reply page-components/links.md]: switched

File: page-components/links.md (hunk 1 above)
Comment (StephanieLevy) [reply page-components/links.md]: sounds good, switched

**Comment by sonnakim** [comment]
@marteki 
Agreed with @Scotchester 's suggestion to add a code example for screenreader-only text. Otherwise 👍

**Comment by marteki** [comment]
@Scotchester @sonnakim Does the added code example work?

**Review by Scotchester — APPROVED** [review]

**Review by StephanieLevy — APPROVED** [review]

**Pull request merged**